soc/ite/ec/it8xxx2/linker.ld

```diff
@@ -179,6 +179,9 @@ SECTIONS
 #ifdef CONFIG_SOC_IT8XXX2_KERNEL_IN_RAM
 			*libkernel.a:*
 #endif
+#ifdef CONFIG_SOC_IT8XXX2_ZEPHYR_IN_RAM
+			*libzephyr.a:*
```

Comment:
Same thing here - can you identify a smaller set of library functions to move into ILM?

Reply:
As above.